code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -195,13 +227,20 @@
  */
 /obj/machinery/computer/pandemic/proc/create_culture_bottle(index)
 	var/id = get_virus_id_by_index(text2num(index))
-	var/datum/disease/advance/adv_disease = SSdisease.archive_diseases[id]
+	var/datum/disease/advance/adv_disease = SSdisease.archive_diseases[id].Copy()
```

Comment:
```suggestion
	var/datum/disease/advance/adv_disease = SSdisease.archive_diseases[id]
```

You shouldn't copy the disease here. That's what's causing linters to fail because this list doesn't have a type. 

Let's shift it down